.github/workflows/mobile-build.yml

```diff
@@ -11,7 +11,7 @@ on:
 
 jobs:
   build-android:
-    if: !github.event.pull_request.draft
```

Comment:
this was invalid syntax

Reply:
looks like this should go back to being broken :P

Reply:
oh hell yeah, didn't know there was an extension!